myapp/main.py

```diff
@@ -0,0 +1,474 @@
+from flask import Flask, render_template, request, flash, redirect, url_for, jsonify, session, make_response
+from flask_sqlalchemy import SQLAlchemy
+import db, schema, time, ast
+try:
+    import simplejson as json
+except (ImportError):
+    import json
+from utility import Generate_lab_id, Generate_row_id, Generate_data_id
+#Initialize an app
+app = Flask(__name__)
+app.debug = True
+app.secret_key = 'A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
+
+
+# 1.Commands
+# Utility method for creating all the tables in the database
+@app.route('/admin/bootstrap')
+def bootstrap():
+    schema.Base.metadata.create_all(db.engine)
+    return "Bootsrapped!"
+
+# Clean all the rows in Lab_rows and Lab_desc
+@app.route('/clean_database', methods=['GET'])
+def clean_database():
```

Comment:
Is this route authenticated? If not it is a huge security vulnerability, since it means that anybody who guesses the route can wipe the database. In general, administrative commands should either be behind an authenticated route that requires the user to be a super user, or available only from the command line (we can assume that if the user has permission to execute commands against the database, then they own the system, whether malicious or not).